products/issue_tracker/backend/models.py

```diff
@@ -0,0 +1,184 @@
+from django.db import models
+from django.utils import timezone
+import uuid
+
+
+class Issue(models.Model):
+    class Status(models.TextChoices):
+        BACKLOG = "backlog", "Backlog"
+        TODO = "todo", "To Do"
+        IN_PROGRESS = "in_progress", "In Progress"
+        TESTING = "testing", "Testing"
+        DONE = "done", "Done"
+
+    class OriginProduct(models.TextChoices):
```

Comment:
Not all of these are products
```suggestion
    class Origin(models.TextChoices):
```